profile/README.md

```diff
@@ -16,18 +16,18 @@ The web version of Oppia currently provides a growing curated set of lessons on
 
 Oppia Web is written using Python, Typescript, and Angular.
 
- - [How do I get started?](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia)  
```

Comment:
I changed this because I think "How do I get started" should link to the wiki which welcomes broad type of contributors. (Not just Coders)